cypress/e2e/users_spec/UsersCreation.cy.ts

```diff
@@ -66,122 +66,87 @@ describe("User Creation", () => {
+    cy.verifyContentPresence("#gender-profile-details", ["Male"]);
+    cy.verifyContentPresence("#averageworkinghour-profile-details", ["14"]);
   });
 
   it("Update the existing user profile Form Mandatory File Error", () => {
-    userCreationPage.clickElementById("user-profile-name");
-    userCreationPage.clickElementById("profile-button");
-    userCreationPage.clickElementById("edit-cancel-profile-button");
-    userCreationPage.clearIntoElementById("firstName");
-    userCreationPage.clearIntoElementById("lastName");
-    userCreationPage.clearIntoElementById("phoneNumber");
-    userCreationPage.clearIntoElementById("altPhoneNumber");
-    userCreationPage.clearIntoElementById("weekly_working_hours");
-    userCreationPage.clickElementById("submit");
+    userCreationPage.clickProfileName();
+    userCreationPage.clickProfileButton();
+    userCreationPage.clickEditCancelProfileButton();
+    userCreationPage.clearFirstName();
+    userCreationPage.clearLastName();
+    userCreationPage.clearPhoneNumber();
+    userCreationPage.clearAltPhoneNumber();
+    userCreationPage.clearWeeklyWorkingHours();
+    cy.submitButton("Update");
     userCreationPage.verifyErrorMessages(EXPECTED_PROFILE_ERROR_MESSAGES);
   });
 
   it("create new user and verify reflection", () => {
-    userCreationPage.clickElementById("addUserButton");
+    userCreationPage.clickAddUserButton();
     userCreationPage.selectFacility("Dummy Shifting Center");
-    userCreationPage.typeIntoElementById("username", username);
-    userCreationPage.typeIntoElementById("password", "Test@123");
+    userCreationPage.typeUserName(username);
+    userCreationPage.typePassword();
     userCreationPage.selectHomeFacility("Dummy Shifting Center");
-    userCreationPage.typeIntoElementById("phone_number", phone_number);
-    userCreationPage.setInputDate("date_of_birth", "25081999");
-    userCreationPage.selectDropdownOption("user_type", "Doctor");
-    userCreationPage.typeIntoElementById("c_password", "Test@123");
-    userCreationPage.typeIntoElementById("qualification", "MBBS");
-    userCreationPage.typeIntoElementById("doctor_experience_commenced_on", "2");
-    userCreationPage.typeIntoElementById(
-      "doctor_medical_council_registration",
-      "123456789",
-    );
-    userCreationPage.typeIntoElementById("first_name", "cypress test");
-    userCreationPage.typeIntoElementById("last_name", "staff user");
-    userCreationPage.typeIntoElementById("email", "test@test.com");
-    userCreationPage.selectDropdownOption("gender", "Male");
-    userCreationPage.selectDropdownOption("state", "Kerala");
-    userCreationPage.selectDropdownOption("district", "Ernakulam");
-    userCreationPage.clickElementById("submit");
-    userCreationPage.verifyNotification("User added successfully");
+    userCreationPage.typePhoneNumber(phone_number);
+    cy.clickAndTypeDate("#date_of_birth", "25081999");
+    cy.clickAndSelectOption("#user_type", "Doctor");
```

Comment:
```suggestion
    cy.clickAndSelectOption("#user_type", "Doctor");
```

- create a new function called selectUserType() and follow general guideline i already gave

Reply:
https://github.com/AnveshNalimela/care_fe/blob/c4569fc16cb7f7f2d29552d660134c1868df9599/cypress/pageobject/Users/UserCreation.ts#L69